src/utils/Dom.ts

```diff
@@ -124,6 +124,18 @@ export class Dom {
     return arr;
   }
 
+  /**
+   * Focuses on an element.
+   * @param preventScroll Whether or not to scroll the page to the focused element.
+   */
+  public focus(preventScroll: boolean) {
+    const { pageXOffset, pageYOffset } = window;
+    this.el.focus();
+    if (preventScroll) {
+      window.scrollTo(pageXOffset, pageYOffset);
```

Comment:
It also feels strange that when a param called `preventScroll` is true, it performs a scroll on line 135.
Maybe `preserveScroll` or `preserveOffset` could be clearer,